### PR TITLE
WIP: Copyright fixes

### PR DESCRIPTION
**Pull request by porteri**
Fixes non-conforming copyright statements in 3 files

### CODE REVIEW

File: src/FV/src/math/lu_implementation.F90 [hunk 1]
```diff
@@ -47,7 +39,8 @@
 !
 
 SUBMODULE (tools_math) lu_implementation
-  IMPLICIT NONE
+    IMPLICIT NONE
+    !! summary: Procedures for matrix LU factorization and solution
```
Comment (zbeekman) [inline src/FV/src/math/lu_implementation.F90]: FORD documentation meta data needs to go immediately after module/submodule declaration

Comment (zbeekman) [reply src/FV/src/math/lu_implementation.F90]: (i.e. move it up before `IMPLICIT NONE`)

File: src/FV/src/math/lu_implementation.F90 [hunk 2]
```diff
@@ -1,18 +1,10 @@
...
-!!     contract # NRC-HQ-60-17-C-0007
-!!
-
-
-
+!
+!     (c) 2019 Guide Star Engineering, LLC
+!     This Software was developed for the US Nuclear Regulatory Commission (US NRC)
+!     under contract "Multi-Dimensional Physics Implementation into Fuel Analysis under
+!     Steady-state and Transients (FAST)", contract # NRC-HQ-60-17-C-0007
```
Comment (zbeekman) [inline src/FV/src/math/lu_implementation.F90]: Changing this back has two problems:

1. It causes breakage with FORD when a pre-processed `.F90` file has a comment with single or double quotes separated by a line break
2. The change causes the copyright data to render nicely in FORD documentation. See https://sourceryinstitute.github.io/MORFEUS-Source/sourcefile/lu_implementation.f90.html for an example

Is there a reason you want to change it back? This contains the same information...

File: src/FV/src/math/lu_implementation.F90 (hunk 1 above)
Comment (zbeekman) [reply src/FV/src/math/lu_implementation.F90]: Ah, I see now... yes, we should *copy* the summary and category meta-data to the module scope too... I wish FORD could inherit this from the parent thing (i.e. the file)

**Comment by porteri** [comment]
@zbeekman I'm good to do whatever you'd like with this PR. I wasn't aware that specific file copyright statements were changed to render FORD better. It threw me for a loop when I saw some were different.

**Pull request closed without merging**